content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -136,6 +137,28 @@ done
 
 Once you successfully upgrade to version v1.6.0, you will be able to view information about engine image versions on the UI.
 
+### Danger Zone Setting Configuration
```

Comment:
I suggest you can highlight the feature and its advantage first, describe the conditions and so on... 

For example


Starting with Longhorn v1.6.0, Longhorn allows you to update danger zone settings immediately, eliminating the need to wait for all volumes to detach. This enhancement is designed to provide users with more flexibility and efficiency in managing their configurations. One of the following conditions that must be met for the immediate effect to occur:

- No Attached Volumes: If there are no volumes attached, the setting change will take effect immediately.
- Engine Image Upgrade (Volume Live Upgrade): During a volume live upgrade, which involves creating a new instance manager pod, the setting change will take effect immediately for the new pod.

If immediate effect conditions aren't met, the setting change is reconciled hourly, taking effect when all volumes detach.

....

Reply:
Modified with the suggestion. Thanks.